### ISSUE TITLE
VPAT\WCAG Documentation

### ISSUE THREAD

**Issue by adamdriscoll** [ironmansoftware/powershell-universal#4618]
### Summary of the new feature / enhancement

We have a customer asking for this and we should have it generally available. We may need to create some additional issues for any changes we need to make. 

https://www.w3.org/WAI/WCAG22/quickref/?versions=2.0
